javascript-and-json/main.js

```diff
@@ -0,0 +1,35 @@
+var library = [
+  {
+    isbn: '978 - 1419735042',
+    title: 'The Rise of Kyoshi',
+    author: 'F. C. Yee, Michael Dante DiMartino'
+  },
+  {
+    isbn: '978 - 0547928227',
+    title: 'The Hobbit',
+    author: 'J. R. R. Tolkien'
+  },
+  {
+    isbn: '978 - 0441013593',
+    title: 'Dune',
+    author: 'Frank Herbert'
+  }
+];
+
+console.log('Library: ', library);
+console.log('Library Type: ', typeof library);
+
+var JSONLibrary = JSON.stringify(library);
```

Comment:
make sure to keep variables lowercase except in special cases. They are typically only used for  `class` definitions.

Reply:
👍 I need to work on my variable naming